.github/workflows/build-docs.yml

```diff
@@ -103,6 +103,10 @@ jobs:
       - name: Install package
         run: pip install -e .
 
+      - name: Install Sphinx extensions
+        run: |
+          pip install sphinxcontrib-googleanalytics
```

Comment:
we should at some point add this to env file, but we will need to ask if other members are happy with this